validator/client/service.go

```diff
@@ -200,6 +200,7 @@ func (v *ValidatorService) Start() {
 		eipImportBlacklistedPublicKeys: slashablePublicKeys,
 		logDutyCountDown:               v.logDutyCountDown,
 		Web3SignerConfig:               v.web3SignerConfig,
+		walletIntializedChannel:        make(chan *wallet.Wallet),
```

Comment:
This channel should be buffered, otherwise it will block on sends if there is no receiver ready. In tests, you can change it to an unbuffered channel, but in runtime we prefer buffered unless there is a specific reason not to
```suggestion
		walletIntializedChannel:        make(chan *wallet.Wallet, 1),
```